ludwig/decoders/llm_decoders.py

```diff
@@ -125,6 +125,8 @@ def forward(self, inputs: List[torch.Tensor], **kwargs):
         }
 
 
+###SXK where is this being used, for the decoder how does one build a custom one, one idea here might
+###be for the user to get a list of already available decoders before they start to build one
```

Comment:
This is being automatically set depending on the output feature type for LLM in the schema definition for that particular output feature type. For e.g. https://github.com/ludwig-ai/ludwig/blob/master/ludwig/schema/features/text_feature.py#L175 for LLM text output features.

Yeah, we'll have this in the docs.